security-config.yml

```diff
@@ -0,0 +1,14 @@
+opendistro_security:
+  dynamic:
+    http:
+      anonymous_auth_enabled: false
+    authc:
+        http_enabled: true
+        transport_enabled: true
+        order: 1
```

Comment:
What does "1" mean here?

Reply:
From the documentation: https://opendistro.github.io/for-elasticsearch-docs/docs/security/configuration/configuration/

> The order makes it possible to chain authentication domains together. The security plugin uses them in the order that you provide.

Even though we have 1 authentication domain, I think we we still need to provide a value for this.

Reply:
Maybe a comment to this effect in the yml file would be good so the meaning is known?

Reply:
I've added a note to the file.